RushMeAdmin/app/routes/index.js

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
I could be wrong, but from what I understand about these changes you have made to the structure, this file would be accessible client-side. This seems like a lot of information is here that should not be, are we sure this is in the right place?

Reply:
You're right; the information shouldn't be available client side. I threw depcrecated code in index.js, unsure where else to put it. I'll commit an empty index.js

Reply:
See [474aa9e](https://github.com/RushMeTeam/RushMeBackend/pull/8/commits/474aa9ec53b605655d932f45befbbeee6068791d)